### DISCUSSION TITLE
Is it supporting monorepo?

### DISCUSSION THREAD

**Discussion by SySagar** [fuma-nama/fumadocs#868] [General]
Currently I am using turborepo and having  a apps/docs files which is this new project. Somehowe it seems that the fuma-ui is broken.  In an indepent next project under any directory , it works fine.

![Capture](https://github.com/user-attachments/assets/f78021c1-9f3b-4a13-89b2-2e2aa3b2fbdb)

**Comment by fuma-nama** [comment]
The FAQ section has answered this

**Reply by SySagar** [reply]
Thanks :)

**Comment by mrzmyr** [comment]
```diff
@import 'tailwindcss';

@import 'fumadocs-ui/css/black.css';
@import 'fumadocs-ui/css/preset.css';

+ @source '../../../node_modules/fumadocs-ui/dist/**/*.js';
```

Adding this line helped for me here :)
